rtos/Thread.h

```diff
@@ -347,6 +347,7 @@ class Thread {
     bool _dynamic_stack;
     Semaphore _join_sem;
     Mutex _mutex;
+    bool _finished;
```

Comment:
I needed to update the PR anyway so I re-ordered this so the size stays the same :astonished:.